controller/controller.go

```diff
@@ -224,6 +245,7 @@ func (c *Controller) Reconcile(req reconcile.Request) (reconcile.Result, error)
 	res, err := c.reconcile(ctx, req)
 	if err != nil {
 		errorGauge.Inc()
+		c.sentry.Capture(ctx, err)
```

Comment:
Thinking about locations where we handle errors, maybe check the boot functions. They are separate from the reconciliation. We could send the boot errors to Sentry as well if that makes sense.

Reply:
added error sending after backoff fails